[CI] CoreWithSecurityClientYamlTestSuiteIT test {yaml=cluster.stats/30_ccs_stats/cross-cluster search stats search} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #4316 / amazonlinux-2_platform-support-aws](https://gradle-enterprise.elastic.co/s/st6dpqtuzy6cu)
- [elasticsearch-pull-request #34589 / part-4](https://gradle-enterprise.elastic.co/s/vgbg7hnzqrhfc)
- [elasticsearch-pull-request #34577 / part-4](https://gradle-enterprise.elastic.co/s/547liuffdak6q)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:core-rest-tests-with-security:yamlRestTest" --tests "org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT" -Dtests.method="test {yaml=cluster.stats/30_ccs_stats/cross-cluster search stats search}" -Dtests.seed=813C6D18EF209C44 -Dtests.locale=sr-Latn -Dtests.timezone=America/Havana -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Byaml%3Dcluster.stats%2F30_ccs_stats%2Fcross-cluster%20search%20stats%20search%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [cluster.stats/30_ccs_stats:149]: field [ccs._search.clusters.cluster_one.took.max] doesn't have a true value
Expected: not "0"
     but: was "0"
```

**Issue Reasons:**
- [main] 3 failures in test test {yaml=cluster.stats/30_ccs_stats/cross-cluster search stats search} (0.5% fail rate in 625 executions)
- [main] 2 failures in step part-4 (0.9% fail rate in 222 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.9% fail rate in 228 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -256,9 +256,6 @@ tests:
 - class: org.elasticsearch.xpack.inference.services.cohere.CohereServiceTests
   method: testInfer_StreamRequest_ErrorResponse
   issue: https://github.com/elastic/elasticsearch/issues/114327
-- class: org.elasticsearch.xpack.security.CoreWithSecurityClientYamlTestSuiteIT
-  method: test {yaml=cluster.stats/30_ccs_stats/cross-cluster search stats search}
-  issue: https://github.com/elastic/elasticsearch/issues/114371
 - class: org.elasticsearch.xpack.inference.services.cohere.CohereServiceTests
   method: testInfer_StreamRequest
   issue: https://github.com/elastic/elasticsearch/issues/114385
@@ -311,9 +308,6 @@ tests:
 - class: org.elasticsearch.packaging.test.EnrollmentProcessTests
   method: test20DockerAutoFormCluster
   issue: https://github.com/elastic/elasticsearch/issues/114885
-- class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
-  method: test {yaml=cluster.stats/30_ccs_stats/cross-cluster search stats search}
-  issue: https://github.com/elastic/elasticsearch/issues/114902
 - class: org.elasticsearch.xpack.inference.DefaultEndPointsIT
   method: testInferDeploysDefaultElser
   issue: https://github.com/elastic/elasticsearch/issues/114913

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.stats/30_ccs_stats.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cluster.stats/30_ccs_stats.yml
@@ -121,10 +121,10 @@
   - is_true: ccs._search.total
   - is_true: ccs._search.success
   - exists: ccs._search.skipped
-  - is_true: ccs._search.took
-  - is_true: ccs._search.took.max
-  - is_true: ccs._search.took.avg
-  - is_true: ccs._search.took.p90
+  - exists: ccs._search.took
+  - exists: ccs._search.took.max
+  - exists: ccs._search.took.avg
+  - exists: ccs._search.took.p90
   - is_true: ccs._search.took_mrt_true
   - exists: ccs._search.took_mrt_true.max
   - exists: ccs._search.took_mrt_true.avg
@@ -145,7 +145,7 @@
   - gte: {ccs._search.clusters.cluster_two.total: 1}
   - exists: ccs._search.clusters.cluster_one.skipped
   - exists: ccs._search.clusters.cluster_two.skipped
-  - is_true: ccs._search.clusters.cluster_one.took
-  - is_true: ccs._search.clusters.cluster_one.took.max
-  - is_true: ccs._search.clusters.cluster_one.took.avg
-  - is_true: ccs._search.clusters.cluster_one.took.p90
+  - exists: ccs._search.clusters.cluster_one.took
+  - exists: ccs._search.clusters.cluster_one.took.max
+  - exists: ccs._search.clusters.cluster_one.took.avg
+  - exists: ccs._search.clusters.cluster_one.took.p90

